Resources/config/services.yml

```diff
@@ -13,10 +13,24 @@ services:
     # and have a tag that allows actions to type-hint services
     MovingImage\Bundle\MICommentsBundle\Controller\:
         resource: '../../Controller'
+        tags: ['controller.service_arguments']
+        public: true
 
     MovingImage\Bundle\MICommentsBundle\Service\CommentsService:
         arguments:
             $autoPublish: '%mi_comments.auto_publish%'
 
     JMS\Serializer: '@jms_serializer'
-    Symfony\Component\Validator\Validator\ValidatorInterface: '@validator'
\ No newline at end of file
+    Symfony\Component\Validator\Validator\ValidatorInterface: '@validator'
+    Doctrine\ORM\EntityManager: '@doctrine.orm.entity_manager'
+
+    MovingImage\Bundle\MICommentsBundle\Admin\CommentAdmin:
+        arguments:
+            - ~
+            - MovingImage\Bundle\MICommentsBundle\Entity\Comment
+            - MovingImage\Bundle\MICommentsBundle\Controller\SonataCommentsController
+        calls:
+            - [setTemplate, ['layout', '@@MIComments/Sonata/layout.html.twig']]
```

Comment:
is this '@@' on purpose?

Reply:
Yes, it is a way of escaping the '@' char, so that it doesn't try resolving a symfony service.